python_scripts/shellies_discovery_gen2.py

```diff
@@ -430,7 +430,7 @@
 )
 TPL_RELAY_TEMPERATURE = "{{{{value_json[^switch:{relay}^].temperature.tC}}}}"
 TPL_SET_BLU_TARGET_TEMPERATURE = "{{{{{{^id^:1,^src^:^{source}^,^method^:^BluTRV.Call^,^params^:{{^id^:{thermostat},^method^:^TRV.SetTarget^,^params^:{{^id^:0,^target_C^:value|round(1)}}}}}}|tojson}}}}"
-TPL_SET_BLU_THERMOSTAT_MODE = "{{%set target=4 if value==^off^ else 21%}}{{{{{{^id^:1,^src^:^{source}^,^method^:^BluTRV.Call^,^params^:{{^id^:{thermostat},^method^:^TRV.SetTarget^,^params^:{{^id^:0,^target_C^:target}}}}}}|to_json}}}}"
+TPL_SET_BLU_THERMOSTAT_MODE = "{{%set enable=true if value==^heat^ else false%}}{{{{{{^id^:1,^src^:^{source}^,^method^:^BluTRV.Call^,^params^:{{^id^:{thermostat},^method^:^Trv.SetConfig^,^params^:{{^id^:0,^config^:{{^enable^:enable}}}}}}}}|to_json}}}}"
```

Comment:
Are you sure that changing `enable` to `false` also changes the valve position to 0?

Reply:
No, this turns off "automatic temperature control".

Reply:
So if the valve is open and I set the hvac mode to off , it will remain open, correct?

Reply:
Yes, the thermostat function will be disabled but the valve position can be changed (I will change the valve position sensor to the number entity).